server/service/src/sync/translations/purchase_order.rs

```diff
@@ -0,0 +1,528 @@
+    "quote_ID": "",
+    "reference": "",
+    "requested_delivery_date": "2021-03-11",
+    "serial_number": 16,
+    "status": "cn",
+    "store_ID": "D77F67339BF8400886D009178F4962E1",
+    "supplier_agent": "",
+    "supplier_discount_amount": 0,
+    "target_months": 0,
+    "total_foreign_currency_expected": 0,
+    "total_local_currency_expected": 0,
+    "user_field_1": "",
+    "user_field_2": ""
+}
+ */
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Serialize)]
+pub struct LegacyPurchaseOrderRow {
+    #[serde(rename = "name_ID")]
+    pub name_id: String,
+    #[serde(rename = "ID")]
+    pub id: String,
+    pub creation_date: NaiveDate,
+    #[serde(default)]
+    pub target_months: Option<f64>,
+    pub status: LegacyPurchaseOrderStatus,
+    #[serde(default)]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub comment: Option<String>,
+    #[serde(default)]
+    #[serde(rename = "currency_ID")]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub currency_id: Option<String>,
+    #[serde(default)]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub reference: Option<String>,
+    #[serde(deserialize_with = "zero_date_as_option")]
+    #[serde(serialize_with = "date_option_to_isostring")]
+    pub confirm_date: Option<NaiveDate>,
+    #[serde(default)]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub created_by: Option<String>,
+    #[serde(rename = "store_ID")]
+    pub store_id: String,
+    #[serde(default)]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub supplier_agent: Option<String>,
+    #[serde(default)]
+    #[serde(rename = "authorizing_officer_1")]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub authorising_officer_1: Option<String>,
+    #[serde(default)]
+    #[serde(rename = "authorizing_officer_2")]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub authorising_officer_2: Option<String>,
+    #[serde(default)]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub freight_conditions: Option<String>,
+    #[serde(default)]
+    #[serde(deserialize_with = "empty_str_as_option")]
+    pub additional_instructions: Option<String>,
+    #[serde(default)]
+    #[serde(deserialize_with = "zero_f64_as_none")]
+    pub agent_commission: Option<f64>,
+    #[serde(default)]
+    #[serde(deserialize_with = "zero_f64_as_none")]
+    pub document_charge: Option<f64>,
+    #[serde(default)]
+    #[serde(deserialize_with = "zero_f64_as_none")]
+    pub communications_charge: Option<f64>,
+    #[serde(default)]
+    pub insurance_charge: Option<f64>,
+    #[serde(default)]
+    #[serde(deserialize_with = "zero_f64_as_none")]
+    pub freight_charge: Option<f64>,
+    #[serde(default)]
+    pub supplier_discount_amount: f64,
+    #[serde(default)]
+    #[serde(rename = "OrderTotalBeforeDiscount")]
+    pub order_total_before_discount: f64,
+    #[serde(default)]
+    #[serde(rename = "OrderTotalAfterDiscount")]
```

Comment:
``` suggestion
    #[serde(rename = "Order_total_before_discount")]
    pub order_total_before_discount: f64,
    #[serde(default)]
    #[serde(rename = "Order_total_after_discount")]
   ``` 
   
   I think these ones got mixed up - but this is the case coming from OG

Reply:
I think these ones got mixed up somehow, but this is what is coming in sync buffer and will fix tests